apps/hash-graph/README.md

```diff
@@ -99,7 +99,7 @@ cargo run -- migrate
 
 ## Generate OpenAPI client
 
-The HASH Graph produces an OpenAPI Spec while running, which can be used to generate the `@hashintel/hash-graph-client` typescript client. In the `/apps/hash-graph` directory run:
+The HASH Graph produces an OpenAPI Spec while running, which can be used to generate the `@local/hash-graph-client` typescript client. In the `/apps/hash-graph` directory run:
 
 ```shell
 cargo make generate-openapi-client
```

Comment:
Above comment is a leading question for this, we'd need to adjust docs to specify the app must be running before this command is useful.

Reply:
(Discussed offline), it does say in the paragraph above that the `OpenAPI` spec is there when the graph is running, so my comment would only be helpful to provide copy-paste lines to run to quickly generate the spec